src/utils/transformers/lower-case.transformer.ts

```diff
@@ -0,0 +1,4 @@
+import { TransformFnParams } from 'class-transformer/types/interfaces';
+
+export const lowerCaseTransformer = (params: TransformFnParams): string =>
+  params.value.toLowerCase().trim();
```

Comment:
Value can be `null` or `undefined` in some cases.

```suggestion
  params.value?.toLowerCase().trim();
```

<img width="1028" alt="image" src="https://user-images.githubusercontent.com/6001723/233156875-2cd94be0-33d2-4400-8d7a-6ee147d0b28f.png">